lib/Launcher.js

```diff
@@ -96,18 +101,32 @@ class Launcher {
       const connectionDelay = options.slowMo || 0;
       const browserWSEndpoint = await waitForWSEndpoint(chromeProcess, options.timeout || 30 * 1000);
       const connection = await Connection.create(browserWSEndpoint, connectionDelay);
-      return new Browser(connection, !!options.ignoreHTTPSErrors, killChrome);
+      browser = new Browser(connection, !!options.ignoreHTTPSErrors, killChrome);
+      return browser;
     } catch (e) {
       killChrome();
       throw e;
     }
 
     function killChrome() {
       helper.removeEventListeners(listeners);
-      if (process.platform === 'win32')
-        childProcess.execSync(`taskkill /pid ${chromeProcess.pid} /T /F`);
-      else
-        process.kill(-chromeProcess.pid);
+      if (temporaryUserDataDir) {
+        // Force kill chrome.
+        if (process.platform === 'win32')
+          childProcess.execSync(`taskkill /pid ${chromeProcess.pid} /T /F`);
+        else
+          process.kill(-chromeProcess.pid, 'SIGKILL');
+        // Attempt to remove temporary profile directory to avoid littering.
+        try {
+          removeSync(temporaryUserDataDir);
```

Comment:
@aslushnikov I copied this from your previous PR. I'm not sure if we need it.

Reply:
That's to handle "process.exit " case.